slasher/rpc/server.go

```diff
@@ -23,9 +23,28 @@ type Server struct {
 
 // IsSlashableAttestation returns an attester slashing if the attestation submitted
 // is a slashable vote.
-func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Attestation) (*ethpb.AttesterSlashing, error) {
-	//TODO(3133): implement attestation validation after attestation store will be merged.
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.IndexedAttestation) (*ethpb.AttesterSlashingResponse, error) {
+	//TODO(#3133): add signature validation
+	if err := ss.SlasherDB.SaveIndexedAttestation(req); err != nil {
```

Comment:
why save this, also is there any chance this might overwrite any previously saved indexed attestation ?

Reply:
there is a check in the save method. this endpoint is used for detection and feeding of data